practices/documentation_testing.md

```diff
@@ -0,0 +1,106 @@
+
+---
+slug:\documentation-testing
+stages:
+  - research
+  - development
+  - maintenance
+aspect:
+	- architecture
+short_description: Documentation testing is a process of improving your existing documentation through feedback. Understanding how to get feedback is crucial for building good documentation and positive developer experience.
+Tags:
+	- technical writing
+	- software documentation
+	- introduction
+	- information
+	- readme
+	- feedback
+	- writing
+	- continuous delivery
+	- value
+	- good developer experience
+keywords:
+  - Developer experience
+  - Tech writing
+  - Getting started tutorial
+  - Testing your Documentation
+  - Tech writing feedback
+covered_by_dx_scanner: false
+---
+# Documentation Testing
+
+**TL;DR** Documentation testing is a process of improving your existing documentation through the Passive Feedback Gathering and Active Testing. Understanding how to work with feedback is crucial for building good documentation and positive developer experience.
+
+## What is Documentation Testing?
+
+As said above, documentation testing is a few processes of continuous feedback gathering and documentation improvements based on feedback. You can imagine it as active listening to your developers. Similarly, as you use testing for your code, you should be testing your docs to provide clear and precise information that contributes to positive developer experience.
```

Comment:
```suggestion
Documentation testing is a few processes of continuous feedback gathering and documentation improvements based on feedback. You can imagine it as active listening to your developers. Similarly, as you use testing for your code, you should be testing your docs to provide clear and precise information that contributes to positive developer experience.
```

I would make the paragraph without a reference to TL;DR because users can read the detailed description without the TL;DR then.